_sass/components/_va-nav-category.scss

```diff
@@ -0,0 +1,58 @@
+// Category navigation AKA "piano keys"
+// pattern. These are the links at the bottom
+// of secondary-level pages.
+
+.va-nav-category {
+  color: $color-gray-dark;
+  list-style: none;
+  margin: 0 !important;
+  padding: 0;
+  
+  @media #{$small} {
+    margin: 0 -0.625rem !important;
+  }
+  
+  &::before, &::after {
+    clear: both;
+    content: " ";
+    display: table; 
+  }
+  
+  li {
+    border-bottom: 1px solid rgba(0,0,0,0.2);
+    display: block;
+    height: auto;
+    margin: 0;
+    width: 100%;
+  }
+  
+  a {
+    color: $color-gray-dark !important;
+    border-radius: 0;
+    border-left: 4px solid rgba(0,0,0,0);
+    border-bottom: none;
+    box-shadow: none;
+    display: block;
+    text-decoration: none;
+    padding: 1em .5em;
+    height: auto;
+    margin: 0;
+    
+    &:hover {
+      background: rgba(0,0,0,0.1);
+      border-left-color: $color-primary;
+      text-decoration: none;
+    }
+  }
+}  
+
+.va-nav-category-title, 
+.va-nav-category h5 {
+  color: #112e51;
```

Comment:
Can we make this color a variable in case other people need to user it somewhere else in the site?